bluedrop-learning-networks/eslint-config-bluedrop
# Release alpha

## Description

Explain the purpose of this pull request and give a brief description of the changes made.

 - What is being modified or created?
    - ex. Modify public FIND /classes to return the number of available seats in the class
    - ex. Add a date range selector UI component
 - Any dependent PRs associated with this one
 - Current behaviour/expected behaviour (if this is a bug fix)
 - How do the changes in this PR affect the overall security of the system? (if applicable)

(Delete any sections below that are not relevant).

## Screenshots/GIFs

Add any relevant images or animations that show the changes made.

## Additional Notes

Include any additional information that is relevant or helpful to review this pull request effectively.

## How to Test

This section should include thorough instructions for the person that is QC testing your PR.

We should be actively trying to break our work! Try to think of use cases that may break your implementation.

## Checklist

- [ ] I have tested this PR locally or on a remote environment.
- [ ] No dependent changes are required to be merged or published, or
